doc/sphinx/documentation/tutorials/small_molecule_tutorial.rst

```diff
@@ -111,9 +104,72 @@ At this stage the reflections may be integrated.  This is done by running
 
 .. code-block:: bash
 
-   dials.integrate refined.refl refined.expt nproc=8
+   dials.integrate refined.refl refined.expt
+
+which will integrate each sequence in sequence, again using all available cores. After integration you can look at the integration shoeboxes with the image viewer, as 
+
+.. code-block:: bash
+  
+  dials.reciprocal_lattice_viewer integrated.refl integrated.expt
+
+
+
+Symmetry Determination
+----------------------
+
+So far the data were processed with a triclinic unit cell, which is usually OK but not ideal for correctly _scaling_ the data. After integration it is usual to determine the symmetry for scaling using
+
+.. code-block:: bash
+
+  dials.symmetry integrated.refl integrated.expt
+
+This will look at the shape of the unit cell and determine the maximum possible symmetry based on the cell parameters, with some tolerance. Each of the possible symmetry operations will be individually tested and scored, and those operations identified as being present will be composed into the point group to be assigned to the data. An attempt is then made to estimate the space group from the presence or absence of axial reflections: this is rather less reliable than the point group determination but also less important for the scaling. After the point group has been determined the reflections will be reindexed automatically to match the correct setting, ensuring that the data are correctly prepared for scaling. 
+
+
+Scaling
+-------
+
+
+In general there is very little which needs to be adjusted in the scaling, as the process is largely automatic. The default model to use for the scaling corrections is "physical" which defines smoothly varying scale factors for the overall intensity and sample decay, with a smoothly varying surface expressed with four or six orders of spherical harmonics for the absorption correction. For organic small molecule crystals the absorption is unlikely to be substantial, but if metals are present or the data are taken with a long wavelength it may be helpful to reduce the strength of the restraints on the absorption surface by setting the absorption level
+
+.. code-block:: bash
```

Comment:
Knowing what default absorption is might make the choice of `medium` level more clear. Assuming the two lines of code correspond to the two crystal cases in text

Reply:
Noted, fixed.